src/templates/layout.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE HTML>
+<html lang="en">
+
+  <!-- <<<<<<< HEAD  -->
```

Comment:
This is a left over of a bad merge. This is not supposed to be there. These can often break your code. Remove this please :)